server/tracedb/datasource/http.go

```diff
@@ -25,7 +25,7 @@ type HttpClient struct {
 }
 
 func NewHttpClient(name string, config *datastoreresource.HttpClientConfig, callback HttpCallback) DataSource {
-	endpoint, _ := url.Parse(config.Url)
+	endpoint, _ := urlx.Parse(config.Url)
```

Comment:
This fixes an annoying but with the URL lib not setting up the expected values when parsing the url

Reply:
What is the main difference between using `url.Parse` and `urlx.Parse`? Is there a way to keep `url.Parse`?

Reply:
Here's the main difference https://github.com/goware/urlx#difference-between-urlx-and-neturl

We are using it here as well https://github.com/kubeshop/tracetest/blob/0bcfa3a3624e29ec8c71d534f3341dd12c315f2d/server/executor/trigger/http.go

Reply:
What is my concern: https://github.com/goware/urlx is a small library without changes for 2 years and we are using only a portion of its capability. 
My thought is that we can transform the `url.URL` return like this lib does.